### ISSUE TITLE
Old Select2 version is breaking Select2 in other plugins

### ISSUE THREAD

**Issue by crabilld** [strangerstudios/pmpro-register-helper#67]
On Line 1108 of pmpro-register-helper.php, the plugin always includes an old version of Select2, which is conflicting with other plugins.

In my case, I'm trying to use a multi-select field in Advanced Custom Fields, which uses Select2. When I add elements to the select field and then save the post, only the first item gets saved. So basically, someone who has Register Helper installed cannot use a Select2 field in ACF. I assume there are other bugs like this in other plugins that use Select2.

This plugin includes version 3.1 of Select2, and ACF is using v3.5. You could simply update the version of Select2, but it would be much better to only enqueue Select2 on registration pages. You have a todo comment that says to add Select2 only "when this is needed", but is there a quicker, simple conditional to just contain this enqueue to the plugin itself? It doesn't need to be the best solution, but just something that takes us 90% of the way there. I'm not familiar enough with the plugin to know the best way of doing this.

**Comment by defunctl** [comment]
I can second that this breaks Avada's Typography fonts on their theme options. Commenting out the following fixes it:
```
wp_enqueue_script('select2', plugins_url('js/select2.js', __FILE__), array( 'jquery' ), '3.1' );
```
![avada-missing-fonts](https://cloud.githubusercontent.com/assets/1066195/15298921/c2cec194-1b56-11e6-955d-991ddd23e587.jpg)

**Comment by defunctl** [comment]
I put in a pull request for this: #71

**Comment by dparker1005** [comment]
Hi @crabilld,

This conflict should have been resolved in PMPro Register Helper v1.3 by @defunctl's pull request here along with other more recent updates to that snippet of code: https://github.com/strangerstudios/pmpro-register-helper/pull/71 (thanks for that!)

As this conflict should now be resolved, I am going to close this issue. If this is a problem that you are still experiencing after updating to the most recent version of PMPro Register Helper, please feel free to create a new issue in this repository so that we can look further into this.
